src/main/java/Car.java

```diff
@@ -0,0 +1,22 @@
+import java.util.Random;
+
+public class Car {
+    private int distance;
+
+    public Car() {
+        this.distance = 0;
+    }
+
+    public int getDistance() {
+        return distance;
+    }
+
+    public void move(int randomNumber) {
+        if (isMove(randomNumber)) distance++;
+    }
+
+    public boolean isMove(int randomNumber) {
+        return randomNumber >= 4;
+    }
```

Comment:
`is` 동사와 `move` 동사가 붙어있으니 약간 어색한 느낌이드네요 😄 
`is` + 형용사로 붙여서 네이밍하면 훨씬 자연스러울 것 같네요 

```suggestion
    public boolean isMovable(int randomNumber) {
        return randomNumber >= 4;
    }
```